full-service/src/json_rpc/v2/api/request.rs

```diff
@@ -57,7 +57,7 @@ pub enum JsonCommandRequest {
         block_version: Option<String>,
         sender_memo_credential_subaddress_index: Option<String>,
         payment_request_id: Option<String>,
-        subaddress_to_spend_from: Option<String>,
+        spend_from_subaddress: Option<String>,
```

Comment:
Renaming from previous PR that introduced these, assuming this feature has not been adopted yet, so no need for backward compatibility